dashboard/dashboard-cluster-info.md

```diff
@@ -0,0 +1,70 @@
+---
+title: TiDB Dashboard Cluster Information Page
+summary: View the running status of TiDB, TiKV, PD, TiFlash components in the entire cluster and the running status of the host on which these components are located.
+category: how-to
+---
+
+# TiDB Dashboard Cluster Information Page
+
+On the cluster information page, you can view the running status of TiDB, TiKV, PD, TiFlash components in the entire cluster and the running status of the host on which these components are located.
+
+## Access the page
+
+You can use one of the following two methods to access the cluster information page:
+
+- After logging into TiDB Dashboard, click **Cluster Info** on the left navigation bar:
```

Comment:
Maybe the `navigation menu` is more suitable? (I'm not sure)

Reply:
I think you are right.